models/volunteer-session.go

```diff
@@ -11,5 +11,5 @@ type VolunteerSession struct {
 	Brigade       Brigade
 	DiscordUserID string        `gorm:"not null"`
 	StartTime     time.Time     `gorm:"not null"`
-	Duration      sql.NullInt32 `gorm:"not null"`
+	Duration      sql.NullInt64
```

Comment:
uint64 data type perhaps? Since Duration cannot be < 0

Reply:
Unfortunately, the golang SQL library doesn't have a NullUInt64 type, only NullInt64. I have to use the SQL data types because until a session is ended, its duration value is null.

Reply:
Ahh I see